Detectors/TRD/workflow/src/TRDTrapSimulatorSpec.cxx

```diff
@@ -296,7 +298,7 @@ void TRDDPLTrapSimulatorTask::run(o2::framework::ProcessingContext& pc)
             for (const auto& newLabel : newLabels) {
               bool alreadyIn = false;
               for (const auto& currLabel : currentLabels) {
-                if (currLabel.compare(newLabel)) {
+                if (currLabel.compare(newLabel) == 1) {
```

Comment:
here too, you don't need to check the `isFake` flags, just use `currLabel == newLabel`

Reply:
Done